memgpt/client/client.py

```diff
@@ -236,7 +236,22 @@ def list_agents(self):
         response = requests.get(f"{self.base_url}/api/agents", headers=self.headers)
         return ListAgentsResponse(**response.json())
 
+    def get_agent_id_from_name(self, agent_name: str) -> uuid:
+        agents = self.list_agents()
+        if agents.num_agents > 0:
+            agent_name = [agent["id"] for agent in agents.agents if agent["name"] == agent_name]
```

Comment:
Maybe it should be named as `agent_ids`. Because of `agent["id"]`

Reply:
Thank you, but the correct correction would be agent_id. The function will return the first matching name. I am assuming that the names of the agents are unique.